contracts/liquidityMining/KyberFairLaunch.sol

```diff
@@ -292,10 +292,29 @@ contract KyberFairLaunch is IKyberFairLaunch, PermissionAdmin, ReentrancyGuard {
 
   /**
    * @dev harvest rewards from multiple pools for the sender
+   *  combine rewards from all pools and only transfer once to save gas
```

Comment:
got extra space before `combine..`

Reply:
kind of my intention for comment which is more than 1 line